code-for-canada/project-thundercat
# [Accessibility] Updating main landmarks

# Description

This PR is refactoring the main landmarks in the whole application. After a follow up with David, I learned that the main landmark shouldn't be in the _ContentContainer_. These main landmarks should be where the current page content is.

![image](https://user-images.githubusercontent.com/23021242/61535061-4754b880-a9ff-11e9-8980-eebbad9e6960.png)

## Type of change

- Code cleanliness or refactor

## Screenshot

(N/A)

# Testing

Manual steps to reproduce this functionality:

1.  Launch NVDA.
2.  Navigate throughout the application and make sure that the main landmarks are where the current page content is.

# Checklist
